eo-maven-plugin/src/main/java/org/eolang/maven/VersionsMojo.java

```diff
@@ -59,13 +58,14 @@
  *
  * @see <a href="https://home.objectionary.com/tags.txt">Tags</a>
  * @since 0.29.6
- * @todo #1602:30min Handle tags that are not in available versions list.
- *  VersionsMojo goes right after OptimizeMojo and replaces all tags with
- *  comparable hashes. EO code may contains tags that are not in available
- *  versions list (see: <a href="https://home.objectionary.com/tags.txt"/>).
- *  We need to catch somehow such versions and throw an exception. Or we can
- *  place the VersionsMojo right after ParseMojo and create new xsl which is
- *  used on optimization step and caches such invalid tags.
+ * @todo #1602:30min Don't rewrite parsed xmir. VersionsMojo is executed right
```

Comment:
@maxonfjvipon Please, fix grammar mistakes in the `todo` - it's hard to read it and understand.